api-compact.yaml

```diff
@@ -12,7 +12,8 @@
 #   /enterprise/api/v1/catalogs/{id}/courses/
 #   /enterprise/api/v1/enterprise-catalogs/
 #   /enterprise/api/v1/enterprise-catalogs/{uuid}/
-#   /enterprise/api/v1/programs/{uuid}/
+#   /enterprise/api/v1/enterprise-catalogs/{uuid}/course-runs/{uuid}/
```

Comment:
Should be `id` rather than `uuid` I believe.

Reply:
Yes. I will update. Thanks.